packages/integrations/useSortable/index.ts

```diff
@@ -70,11 +80,20 @@ export function useSortable<T>(
       return sortable?.option(name)
   }
 
+  watch(() => toValue(el), (value) => {
+    if (isActive.value)
+      stop()
+
+    if (value)
+      start()
+  }, { flush: 'post' })
```

Comment:
Thank you for your PR: Please note https://vueuse.org/guidelines.html#watch-options